test/Microsoft.AspNet.Localization.Tests/AccessLanguageHeaderRequestCultureProviderTest.cs

```diff
@@ -106,5 +106,38 @@ public class AccessLanguageHeaderRequestCultureProviderTest
                 var response = await client.GetAsync(string.Empty);
             }
         }
+        
+        [Fact]
+        public async void OmitDefaultRequestCultureShouldNotThrowNullReferenceException_And_ShouldGetTheRightCulture()
```

Comment:
In fact support both ```SupportedCultures``` & ```SupportedUICultures``` cause the error, I tested in my end. If you notice not ```OmitDefaultRequestCultureShouldNotThrowNullReferenceException_And_ShouldGetTheRightCulture``` is the only test that support both of them